Python/di_sensors/easy_light_color_sensor.py

```diff
@@ -151,16 +151,17 @@ def safe_raw_colors(self):
 
         The colors detected vary depending on the lighting conditions of the nearby environment.
 
-        :returns: The RGBA values from the sensor. RGBA = Red, Green, Blue, Alpha (or Clear). Range of each element is between **0** and **1**.
+        :returns: The RGBA values from the sensor. RGBA = Red, Green, Blue, Alpha (or Clear). Range of each element is between **0** and **1**. -1 means an error occured.
         :rtype: tuple(float,float,float,float)
```

Comment:
Also, we could structure it as: `:rtype: tuple(float,float,float,float) or just float in case an error occurs`.